### ISSUE TITLE
TestSyncClusterWithoutVersioning does not complete [Windows]

### ISSUE THREAD

**Issue by Zillode** [syncthing/syncthing#1769]
http://build.syncthing.net/job/syncthing-windows-integration/116/console

```
=== RUN TestSyncClusterWithoutVersioning
2015/05/06 22:57:57 Testing with numFiles=100, fileSizeExp=20, iterations=3
2015/05/06 22:57:57 Cleaning...
2015/05/06 22:57:57 Generating files...
2015/05/06 22:58:06 Waiting for startup...
2015/05/06 22:58:06 Forcing rescan...
2015/05/06 22:58:07 Syncing...
2015/05/06 22:58:10 I6KAH76: JMFJCXB at 16%
2015/05/06 22:58:13 I6KAH76: 373HSRP at 15%
2015/05/06 22:58:16 I6KAH76: 373HSRP at 46%
2015/05/06 22:58:19 I6KAH76: JMFJCXB at 78%
2015/05/06 22:58:22 JMFJCXB: I6KAH76 at 73%
2015/05/06 22:58:25 JMFJCXB: I6KAH76 at 73%
2015/05/06 22:58:28 JMFJCXB: I6KAH76 at 73%
2015/05/06 22:58:31 JMFJCXB: I6KAH76 at 73%
2015/05/06 22:58:34 JMFJCXB: I6KAH76 at 73%
2015/05/06 22:58:37 JMFJCXB: I6KAH76 at 73%
2015/05/06 22:58:40 JMFJCXB: I6KAH76 at 73%
2015/05/06 22:58:43 JMFJCXB: I6KAH76 at 73%
2015/05/06 22:58:46 JMFJCXB: I6KAH76 at 73%
2015/05/06 22:58:49 JMFJCXB: I6KAH76 at 73%
2015/05/06 22:58:53 JMFJCXB: I6KAH76 at 73%
2015/05/06 22:58:56 JMFJCXB: I6KAH76 at 73%
```

**Comment by calmh** [comment]
Needs logs; should probably save the test/logs directory as artifacts...

**Comment by Zillode** [comment]
(probably dup of https://github.com/syncthing/syncthing/issues/1765)

**Comment by Zillode** [comment]
Nope,
S1 https://gist.githubusercontent.com/Zillode/3ae5972a03f45e48ef62/raw/25fbfcd0afdfe7235d0d327959a73d721bde6c81/gistfile1.txt
S2 https://gist.githubusercontent.com/Zillode/ac77c746e3caaa996013/raw/907b074ff6e3940bf0c742ef06bf0afdba099897/gistfile1.txt
S3 https://gist.githubusercontent.com/Zillode/d0c314a4a2d333d2ffe1/raw/4385ce0c14bfa47ffc32a8f7889d677725ac8906/gistfile1.txt

```
[I6KAH] 2015/05/08 12:02:48.607917 model.go:689: INFO: Connection to JMFJCXB-GZDE4BN-OCJE3VF-65GYZNU-AIVJRET-3J6HMRQ-AUQIGJO-FKNHMQU closed: WSARecv tcp 127.0.0.1:22001: An existing connection was forcibly closed by the remote host.
```
```
[JMFJC] 2015/05/08 11:10:09.554103 connections.go:306: INFO: TLS handshake: WSARecv tcp 127.0.0.1:63919: An existing connection was forcibly closed by the remote host.
```
```
[373HS] 2015/05/08 12:02:48.618168 model.go:689: INFO: Connection to JMFJCXB-GZDE4BN-OCJE3VF-65GYZNU-AIVJRET-3J6HMRQ-AUQIGJO-FKNHMQU closed: WSARecv tcp 127.0.0.1:63925: An existing connection was forcibly closed by the remote host.
```
